src/Engine/EmitMSIL/Replication.cs

```diff
@@ -587,7 +566,7 @@ private static IList<CLRStackValue> getSubParameters(CLRStackValue o)
 
             if (o.IsEnumerable)
             {
-                return o.Value as IList<CLRStackValue>;
+                return o.Value as IList<CLRStackValue> ?? new List<CLRStackValue>() { o };
```

Comment:
I am trying to figure out why `o.Value` would not be `IList<CLRStackValue>`
The Marshaler should always convert enumerables to `IList<CLRStackValue>` I think...

Reply:
Well, in this particular function I still haven't encountered the case where `o.Value` is something other than `IList<CLRStackValue>` so I guess I can revert this but definitely in the other places I needed to check if the `Value` property was something other than `IList<CLRStackValue>`. 

Did your question pertain to just this function or all such changes in general?